pvr.octonet/resources/language/resource.language.en_gb/strings.po

```diff
@@ -27,3 +27,33 @@ msgstr ""
 msgctxt "#30001"
 msgid "Could not load chanellist"
 msgstr ""
+
+msgctxt "#30002"
+msgid "Open stream using"
+msgstr ""
+
+msgctxt "#30003"
+msgid "Live stream API"
+msgstr ""
+
+msgctxt "#30004"
+msgid "Kodi's inputstream"
+msgstr ""
+
+msgctxt "#30005"
+msgid "inputstream.ffmpegdirect"
+msgstr ""
+
+msgctxt "#30006"
+msgid "inputstream.rtmp"
+msgstr ""
+
+#empty strings from id 30007 to 30599
+
+msgctxt "#30600"
+msgid "The address of the octonet server. Note that http:// prefix should not be entered, just the address"
```

Comment:
maybe change this to `The IP/address of the octonet server. Note that in case of an address, the protocol (http://) has to be omitted.`

Reply:
Done